server_test.go

```diff
@@ -799,19 +799,6 @@ func TestShutdownUDP(t *testing.T) {
 	}
 }
 
-func TestShutdownUDPWithContext(t *testing.T) {
```

Comment:
Why removing this test ?

Reply:
@fturib A few reasons, it was UDP only and didn't have a TCP and TLS mirror; and that I don't think it really added much over `TestInProgressQueriesAtShutdown*` which really exercised the shutdown code. What was your intent behind it?

Reply:
This code is testing the shutdown when there is no dns activity, but in a case where the deadline of context was > than readTimeout.
I guess that is also tested in TestShutdownUDP.